assets/styl/_mixins/_contrast-check.styl

```diff
@@ -0,0 +1,19 @@
+// Contrast-check
+//
+// return true if constast is OK
+//
+// return false and throw a warning message if it’s not
+//
+// $foreground-color - text color to check
+// $background-color - background color to compare. default $color-white
+// $min              - minimum contrast ratio. default $contrast
+//
+// Styleguide Mixins.Contrast-check
+contrast-check($foreground-color, $background-color = $color-white, $min = $contrast)
```

Comment:
I usually prefix mixins with a `_`, and use camelCase for naming (see [an example here](https://github.com/20minutes/colette/blob/master/assets/styl/_layout/_grid.styl#L37))